overlay/usr/lib/inithooks/bin/domain-controller.py

```diff
@@ -318,24 +412,42 @@ def main():
 
         if not admin_password:
             d = Dialog('TurnKey Linux - First boot configuration')
+            if create:
+                server_status = 'new'
+            else:
+                server_status = 'existing'
```

Comment:
also could be simplified with ternary: `server_status = 'new' if create else 'existing'`